jdaviz/configs/imviz/helper.py

```diff
@@ -26,6 +41,157 @@ def load_data(self, data, parser_reference=None, **kwargs):
 
         self.app.load_data(filepath, parser_reference=parser_reference, **kwargs)
 
+    # astrowidgets ducktyping (MVP)
+
+    def center_on(self, point):
+        raise NotImplementedError
+
+    def offset_to(self, dx, dy, skycoord_offset=False):
+        raise NotImplementedError
+
+    @property
+    def zoom_level(self):
+        raise NotImplementedError
+
+    @zoom_level.setter
+    def zoom_level(self, val):
+        raise NotImplementedError
+
+    def zoom(self, val):
+        raise NotImplementedError
+
+    @property
+    def stretch_options(self):
+        raise NotImplementedError
+
+    @property
+    def stretch(self):
+        raise NotImplementedError
+
+    @stretch.setter
+    def stretch(self, val):
+        raise NotImplementedError
+
+    @property
+    def autocut_options(self):
+        raise NotImplementedError
+
+    @property
+    def cuts(self):
+        raise NotImplementedError
+
+    @cuts.setter
+    def cuts(self, val):
+        raise NotImplementedError
+
+    @property
+    def colormap_options(self):
+        raise NotImplementedError
+
+    def set_colormap(self, cmap):
+        raise NotImplementedError
+
+    @property
+    def click_center(self):
+        return False  # No way to do this right now
+
+    @click_center.setter
+    def click_center(self, val):
+        raise NotImplementedError
+
+    @property
+    def click_drag(self):
+        return False  # No way to do this right now
+
+    @click_drag.setter
+    def click_drag(self, value):
+        raise NotImplementedError
+
+    @property
+    def scroll_pan(self):
+        return False  # No way to do this right now
+
+    @scroll_pan.setter
+    def scroll_pan(self, value):
+        raise NotImplementedError
+
+    def save(self, filename):
+        raise NotImplementedError
+
+    # astrowidgets ducktyping (not MVP)
+    #
+    # NOTES:
+    # * logger is excluded because it is too Ginga-specific.
+    # * cursor is excluded because coordinate info is in a plugin.
+
+    @property
+    def image_width(self):
+        raise NotImplementedError
+
+    @image_width.setter
+    def image_width(self, value):
+        raise NotImplementedError
+
+    @property
+    def image_height(self):
+        raise NotImplementedError
+
+    @image_height.setter
+    def image_height(self, value):
+        raise NotImplementedError
+
+    @property
+    def pixel_offset(self):
+        return 0  # No way to customize right now
+
+    # Loaders
+
+    def load_fits(self, fitsorfn, numhdu=None, memmap=None):
+        raise NotImplementedError
+
+    def load_nddata(self, nddata):
+        raise NotImplementedError
+
+    def load_array(self, arr):
+        raise NotImplementedError
```

Comment:
I think this is MVP in the sense that it's part of the existing `load_data`, right?  I.e., these could be trivial wrappers into that?

Reply:
I think we debated whether having `load_xxx` in addition to `imviz.load_data` would be too confusing.

[PEP 20](https://www.python.org/dev/peps/pep-0020/): `There should be one-- and preferably only one --obvious way to do it.`